docs/api/snowflake/vector-data/Function.md

```diff
@@ -1486,6 +1486,35 @@ Output:
 false
 ```
 
+## ST_IsValidDetail
+
+Introduction: Returns a row, containing a boolean `valid` stating if a geometry is valid, a string `reason` stating why it is invalid and a geometry `location` pointing out where it is invalid.
```

Comment:
This is a Snowflake table function. The doc example is not correct

Reply:
The usage of Snowflake table function is different from others. See ST_SubdivideExplode as an example: https://sedona.apache.org/1.6.0/api/snowflake/vector-data/Function/#st_subdivideexplode

```
SELECT Sedona.ST_AsText(GEOM)
FROM table(Sedona.ST_SubDivideExplode(Sedona.ST_GeomFromText('LINESTRING(0 0, 85 85, 100 100, 120 120, 21 21, 10 10, 5 5)'), 5));
```